### DISCUSSION TITLE
Firebird 2.5 | 3  -> Firebird 5

### DISCUSSION THREAD

**Discussion by nLeonardt95** [FirebirdSQL/firebird#7894] [General]
Is it possible to restore a Firebird 2.5 or 3 backup with Firebird 5 without any restrictions?
Or is there anything else I should keep in mind?

**Comment by mrotteveel** [comment]
Restoring a backup from older versions to Firebird 5 should work. Though especially coming from Firebird 2.5, it might be worthwhile to recreate the database, taking into account the new data types introduced in Firebird 3.0 and 4.0, and pumping over the data from the old DB to the new.

**Reply by mrotteveel** [reply]
As an aside, I transferred your question to "Discussions", as we use the issue tracker only for bug reports and feature requests.

Though we have "Discussions" here, in general, I recommend asking questions like this on [firebird-devel](https://groups.google.com/g/firebird-devel) (for unreleased versions like Firebird 5), or [firebird-support](https://groups.google.com/g/firebird-support) (for released versions) as you'll likely get more eyes on it there
